02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/03 Scikit-Learn/04 Build Models.html

```diff
@@ -0,0 +1,38 @@
+<p>In this example, build a support vector regression prediction model using the following features and labels:</p>
+
+<table class="qc-table table">
+    <thead>
+        <tr>
+            <th>Data Category</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Features</td>
+            <td>Daily percent change of the open, high, low, close, and volume of the SPY over the last 5 days</td>
+        </tr>
+        <tr>
+            <td>Labels</td>
+            <td>Daily percent return of the SPY over the next day</td>
+        </tr>
+    </tbody>
+</table>
+
+<p>The following image shows the time difference between the features and labels:</p>
+<img class="docs-image" src="https://cdn.quantconnect.com/i/tu/ml-keras-function.png">
+
+<p>Follow these steps to create a method to build the model:</p>
```

Comment:
Condense these two steps into 1 line of code.

Then change the "Follow these steps ... " to "To build the model, call the GridSearchCV ..."